hr_expense_invoice/models/hr_expense_sheet.py

```diff
@@ -112,3 +143,30 @@ def action_view_invoices(self):
             action["view_mode"] = "tree,form"
             action["domain"] = [("id", "in", invoice_ids)]
         return action
+
+    def _compute_state(self):
```

Comment:
Missing `api.depends`, it could be an empty one.

Reply:
done

Reply:
I don't think it's needed if you are overriding an existing one.